package.json

```diff
@@ -2,29 +2,42 @@
   "name": "crud-car-registration",
   "version": "1.0.0",
   "description": "",
-  "main": "./src/server.ts",
+  "main": "./src/index.ts",
   "scripts": {
-    "start": "nodemon src/server.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon --exec \"ts-node-dev -r tsconfig-paths/register\" src/index.ts",
```

Comment:
great! but 2 things:

1. when you working with a specific branches that name is ModelMethods, you need to work only with model methods.
2. Please use this command on the Aicon project.